backend/lib/seeds/test_survey_questions.csv

```diff
@@ -1,18 +1,26 @@
-7,"If yes, please confirm your name",1,""
-8,"If yes, please confirm your email address",2,""
-9,"If yes, please confirm your phone number",3,""
-10,"So we can report back on your potential savings, what type of heating system do you have? (Across the whole home, not just in individual rooms)",0,"Forced hot air – vents/Radiators – water/Radiators – steam/Baseboard – water/Baseboard – electric/Other"
-11,"If other, please specify",1,""
-12,"What type of heating FUEL do you use for that heating system?",0,"Oil/Gas/Electric (for baseboard)/Other"
-13,"If other, please specify",1,""
-14,"We expect to have “heat pump coaches” available to walk through the process with you (volunteers, who are not affiliated with any installer or brand). Would you like to know more about how to get free, personal help from a heat pump coach?",0,"Yes/No"
-15,"If yes, please confirm your name",1,""
-16,"If yes, please confirm your email address",2,""
-17,"If yes, please confirm your phone number",3,""
+1,"Are you the homeowner?",0,"Yes/No (can we leave this info with you for the homeowner?)"
+2,"Have you heard of heat pumps for home heating and cooling?",0,"Yes/No"
+3,"Here are some common benefits associated with heat pumps - are any of these important to you? Which is your top priority?",0,"Save money on energy bills/Having cooling as well as heating/Quieter than AC/Reducing use of fossil fuels & climate change/I like heat pumps more than my current HVAC system/If I like heat pumps more than my current HVAC system"
+4,"Have you or would you consider(ed) heat pumps for home heating? Or for cooling?",0,"Yes, for heating, not for cooling/Yes for cooling, not for heating/Yes, for both/No - heard of them but have not considered them for either/No answer - Not Available"
+5,"If you didn't consider heat pumps for either or both heating/cooling, why not?",0,"Didn't know they could do heating & cooling/Only need help with one: heating or cooling/Didn't think heat pumps could be efficient & cost effective doing heating & cooling/Didn't know how to get more info & begin the installation process/Too expensive/Home not compatible with heat pumps/Other"
+6,"If you did consider heat pumps for either or both heating & cooling, how far have you taken your interest in heat pumps?",0,"Basic research & Looked at a website/Decided on a plan/Contacted installers & Got quotes/Installed a heat pump/Other"
+7,"If you have considered but have not installed a heat pump yet, what factors have prevented you from moving forward? Was it...",0,"Didn't understand the options, details & next steps/Too expensive/Complexity, too much time, or hassle to do paperwork/Installation concerns & Bad installer experience/Home not compatible with heat pumps/Other"
+8,"Have you heard of Mass Save [wait for reaction]... the energy efficiency program?",0,"Yes/No"
```

Comment:
i'm pretty sure this will be the public survey too, right? if yes, probably don't want "wait for reaction"

Reply:
Removed.